packages/turbo-r2-archive/src/routes.ts

```diff
@@ -0,0 +1,122 @@
+import { zValidator } from "@hono/zod-validator";
+import { Hono } from "hono";
+import { bearerAuth } from "hono/bearer-auth";
+import { cors } from "hono/cors";
+import { HTTPException } from "hono/http-exception";
+import { z } from "zod";
+import { bustOldCache } from "./autoCacheBust";
+
+export const router = new Hono<{ Bindings: Env }>();
+
+const paramValidator = z.object({
+	artifactID: z.string(),
+	teamID: z.string().optional(),
+});
+const queryValidator = z.object({
+	teamId: z.string().optional(),
+	slug: z.string().optional(),
+});
+
+router.onError((error, c) => {
+	if (error instanceof HTTPException) {
+		return error.getResponse();
+	}
+	return c.json({ ...error }, 500);
```

Comment:
Can we add Sentry integration here?

Reply:
Maybe, I am thinking it could be more useful to pipe into something like Grafana or something similar, if it has the metrics I think it does (I havent checked yet)

Reply:
oh nvm this is the error section 😆

Reply:
Im cool with doing this as a follow up. Much like the other stuff I mentioned when I confused this comment with the event endpoint lol